src/lsm/tree.zig

```diff
@@ -933,18 +929,19 @@ pub fn TreeType(comptime TreeTable: type, comptime Storage: type, comptime tree_
             tree.manifest.compact(compact_manifest_callback);
         }
 
+        /// Called after the fourth/last beat.
```

Comment:
```suggestion
        /// Called after the last beat of a compaction measure.
```

Since the beats in a measure are typically > 4.